R/class-KPartiteNetwork.R

```diff
@@ -116,4 +116,75 @@ KPartiteNetwork <- setClass("KPartiteNetwork",
     partitions = Partitions()
   ),
   validity = check_kpartite_network
-)
\ No newline at end of file
+)
+
```

Comment:
i am late on this comment so it's fine to include later, but we should have check_kpartite_network check the links as well to make sure each link's source and target are from different partitions.